voxbox/Rakefile

```diff
@@ -4,3 +4,19 @@ require 'voxpupuli/release/rake_tasks'
 require 'ra10ke'
 
 Ra10ke::RakeTask.new
+
+# re-enable by puppetlabs disabled checks from puppetlabs/spec_helper
+# see: https://github.com/puppetlabs/puppetlabs_spec_helper/blob/70de49db0a242f83e9ff39ea8a03c830339f8368/lib/puppetlabs_spec_helper/rake_tasks.rb#L169-L177
```

Comment:
I think it will confuse a lot of people when the container provided by Vox Pupuli uses different puppet-lint checks compared to our normale `lint` rake task. I don't think that's a good idea.

Reply:
hmja, but this is 💩 we have hardcoded disabled checks. and i like them to be enabled. like the normal behavior from puppet-lint. if we want to keep this behavior, maybe i can add a new rake task which has my changes in it 🤔

Reply:
hmm okay, then everybody has to think about getting all checks and know about that there might be some check disabled and force this upon all modules the one is maintaining.... okay :(

Reply:
i wrapped it into a separate task, i don't have the patience to go and fight.... eh, discuss this on a puppetlabs module...